client/src/modules/user/UserProfile/EmailNotifications.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { PureComponent } from 'react';
+import { injectIntl, FormattedMessage } from 'react-intl';
+
+import AlertBox from 'common/components/AlertBox';
+import SwitchButton from 'common/components/SwitchButton';
+import { IntlPropType } from 'common/constants/propTypes';
+import { saveEmailNotificationSettings } from '../model/actions';
+import { MessageType } from 'common/constants/enums';
+import './EmailNotification.scss';
+
+class EmailNotifications extends PureComponent {
+  state = {
+    error: false,
+    never: false,
+    weekly: true
+  };
+
+  setWeekly = () =>
+    this.setState(
+      { weekly: true, never: false, error: false },
+      this.updateNotificationSettings
+    );
+
+  setNever = () =>
+    this.setState(
+      { never: true, weekly: false, error: false },
+      this.updateNotificationSettings
+    );
+
+  updateNotificationSettings = () => {
+    const { weekly, never } = this.state;
+    const settings = {
+      never,
+      weekly
+    };
+
+    try {
+      saveEmailNotificationSettings(settings);
+    } catch {
+      this.setState({ error: true });
+    }
+  };
+
+  render() {
+    const {
+      intl: { formatMessage }
+    } = this.props;
+    const { error, never, weekly } = this.state;
+
+    return (
+      <section className="email-notifications">
+        {error && (
+          <AlertBox type={MessageType.ERROR}>
+            <FormattedMessage id="email-notifications.save-settings-failure" />
+            <span>&nbsp;</span>
+            <FormattedMessage id="common.try-again" />
+          </AlertBox>
+        )}
+        <h2 className="email-notifications__heading">
+          <FormattedMessage id="email.notification.heading" />
+        </h2>
+        <ul className="email-notifications__list">
+          <li className="email-notifications__list-item">
+            <SwitchButton
```

Comment:
Because  these options are separable I think it will be better to use `<radio />` or If we would have more options `<selecy />` tag